src/transformers/generation/logits_process.py

```diff
@@ -1714,36 +1720,6 @@ class ForceTokensLogitsProcessor(LogitsProcessor):
     indices that will be forced before generation. The processor will set their log probs to `inf` so that they are
     sampled at their corresponding index. Originally created for
     [Whisper](https://huggingface.co/docs/transformers/model_doc/whisper).
-
-    Examples:
```

Comment:
This processor is going to be removed in `v4.40`, so I didn't want to spend time fixing the test :D

Reply:
cheeky :D